wgpu/src/image.rs

```diff
@@ -75,24 +82,25 @@ impl Pipeline {
             transform: Transformation::identity().into(),
         };
 
-        let uniforms_buffer = device.create_buffer_with_data(
-            uniforms.as_bytes(),
-            wgpu::BufferUsage::UNIFORM | wgpu::BufferUsage::COPY_DST,
-        );
+        let uniforms_buffer =
+            device.create_buffer_init(&wgpu::util::BufferInitDescriptor {
+                label: None,
```

Comment:
nit: would be nice to sprinkle meaningful labels to the internal objects. Users who debug this would appreciate!